index.html

```diff
@@ -5401,7 +5406,7 @@ <h5>Presentational Roles Conflict Resolution</h5>
 		<div class="role" id="range">
 			<rdef>range</rdef>
 			<div class="role-description">
-				<p>An input representing a range of values that can be set by the user.</p>
+				<p>An element representing a range of values that can be set by the user.</p>
```

Comment:
@joanmarie, thought of one other aspect of this definition that will not be accurate after meter is included ... this might also be true for progressbar: the user can not set the value.

Reply:
Oops. Only just saw this today. I just removed "that can be set by the user" from the definition of range in both master and stable as it's an editorial change. Thanks for catching that!